src/android/RogerthatPlugin.java

```diff
@@ -622,6 +622,25 @@ public void populateResult(Boolean valid, JSONObject obj) throws JSONException {
         mActivity.getActionScreenUtils().verifySignedPayload(args, sc);
     }
 
+    private void listKeyPairs(final CallbackContext callbackContext, final JSONObject args) {
+        SecurityCallback<List<Map<String, String>>> sc = new SecurityCallback<List<Map<String, String>>>(callbackContext) {
+            @Override
+            public void populateResult(List<Map<String, String>> keyPairs, JSONObject obj) throws JSONException {
+                JSONArray array = new JSONArray();
```

Comment:
It should work without this stuff by just supplying the `keyPairs` variable to obj.put but that would give some weird kind of object representation of the object instead of a proper json string